lessons/07-mcp/README.md

```diff
@@ -0,0 +1,482 @@
+# MCP, Model Context Protocol
```

Comment:
To be consistent with all of the other chapters this should be:

Lesson 7: Model Context Protocol (MCP)

But, is this lesson 7 or lesson 6? Looks like it jumps from 5 to 7.